app/models/pacbio/run.rb

```diff
@@ -45,8 +45,20 @@ class Run < ApplicationRecord
     attr_reader :plates_attributes
 
     # if comments are nil this blows up so add try.
+
+    def generate_comment
+      comment = "#{comments} ".presence || ''
```

Comment:
I am not sure you need to set comments to an emty string here as you are already doing this in comments accessor.